client/src/components/imageList.jsx

```diff
@@ -1,7 +1,89 @@
 import React from 'react';
+import style from '../../styles.css.js';
+import RoomTitle from './roomTitle.jsx';
+
 
 var ImageList = (props) => {
-  return ( <div> Goodbye </div> )
+  let currentRoomIndex = 0
+  console.log('room: ', props.rooms);
+  let roomImgCount = 0;
+  let i = 0;
+
+  let checkInit = false;
+
+  // let newRoomTitle = (room) => (
+  //   <div className='room-name'>
+  //     <h1> TEST </h1>
+  //   </div>
+  // );
+
+  return props.imageUrls.map(url => {
+    let room = props.rooms[currentRoomIndex];
+    console.log('room: ', props.rooms);
+    if (checkInit === false) {
+      checkInit = true;
+      roomImgCount += 1;
+      i += 1;
+      let roomName = room.name.slice(0, -1).split('_').join(' ');
+
+
+      return (
+        <div style={style.listStyle}>
+          <table>
+            <tr className='room-title'>
+              <th >{roomName}</th>
+            </tr>
+            {/* <tr style={style.listStyle} className='home-explore-photos' id='room-images-list'>
+              <td>                
+                <img style={style.listImageStyle} src={url}></img>
+              </td>  
+            </tr> */}
+            <td style={style.listStyle} className='home-explore-photos' id='room-images-list'>                
+                <img style={style.listImageStyle} src={url}></img>
+            </td>
+          </table>
+        </div>
+      )
+    } else if (roomImgCount >= room.numImages) {
```

Comment:
It took me a while to follow the if conditionals, it would be better to simplify it if it is possible.